### DISCUSSION TITLE
Adding custom breakpoint

### DISCUSSION THREAD

**Discussion by Bersaelor** [swordev/suid#182] [Q&A]
Hello, I'd like to add a custom breakpoint like:

```tsx
const isMobile = `@media only screen and (max-width: 600px), (max-height: 600px)`;
const xs = useMediaQuery(isMobile);
// not just const xs = useMediaQuery(useTheme().breakpoints.down('sm'));
```
so I can match phones in both landscape and portrait.
Now, I also like to use `<Grid item xs={12} sm={3}>` when working in MUI, which of course then triggers at different screen sizes then the above simple query would.

Did anyone encounter similar situations?
Do I have to do the grid-sizing without the convenient breakpoints now?

**Comment by juanrgm** [comment]
https://github.com/swordev/suid/issues/170#issuecomment-1416710989
